test/support/utils.rb

```diff
@@ -16,6 +16,12 @@ def assert_line_num(expected)
       })
     end
 
+    def assert_line_text(expected)
+      @queue.push(Proc.new { |result|
+        assert_match(expected, result)
+      })
+    end
```

Comment:
To prevent this file from getting large, we need to create a class for assertion.
I will do that in new PR 👍

Reply:
sounds good 👍 
I guess you mean `module` though? generally the assertion helpers are defined in assertion modules, which are included in the test classes (e.g. [irb's assertion module](https://github.com/ruby/irb/blob/9c19899fe5/test/lib/test/unit/assertions.rb)).

Reply:
Oh, that's right. Thank you!